### PR TITLE
Personal group invites replacing "Add members"

### PR DESCRIPTION
**Pull request by megrogan**

### CODE REVIEW

File: frontend/app/src/components/home/groupdetails/Members.svelte [hunk 1]
```diff
@@ -1,37 +1,73 @@
...
     $: me = knownUsers.find((u) => u.userId === userId);
-    $: others = knownUsers
-        .filter((u) => matchesSearch(searchTerm, u) && u.userId !== userId)
+    $: fullMembers = knownUsers
+        .filter(
+            (u) =>
+                matchesSearch(searchTerm, u) &&
+                u.userId !== userId &&
+                !$blocked.has(u.userId) &&
+                !$invited.has(u.userId)
```
Comment (megrogan) [inline frontend/app/src/components/home/groupdetails/Members.svelte:39]: @julianjelfs I don't think this is necessary is it? "currentChatMembers" won't contain blocked or invited.
```
    !$blocked.has(u.userId) &&
    !$invited.has(u.userId)
```

Comment (julianjelfs) [reply frontend/app/src/components/home/groupdetails/Members.svelte:39]: Quite possibly not necessary. I had to stop kind of in the middle so I'm not really sure.

File: frontend/app/src/components/home/groupdetails/InviteUsersWithLink.svelte [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+<div class="link-enabled">
+    <div class="link">{link}</div>
+    <div class="qr-wrapper">
+        <div class="qr">
+            <QR text={link} />
+        </div>
+    </div>
+    <div class="message">
+        {$_("group.invite.shareMessage") +
```
Comment (julianjelfs) [inline frontend/app/src/components/home/groupdetails/InviteUsersWithLink.svelte:43]: These two messages don't exist in the en.json file

Comment (megrogan) [reply frontend/app/src/components/home/groupdetails/InviteUsersWithLink.svelte]: Think this must have got broken after a merge conflict

**Review by julianjelfs — APPROVED** [review]

**Pull request merged**